modules/virt-supported-cluster-version.adoc

```diff
@@ -7,3 +7,8 @@
 = Supported cluster versions for {VirtProductName}
 
 {VirtProductName} {VirtVersion} is supported for use on {product-title} {product-version} clusters. To use the latest z-stream release of {VirtProductName}, you must first upgrade to the latest version of {product-title}.
+
+[NOTE]
+====
+Virtualization is currently available on x86-64 CPUs. Arm-based nodes are not yet supported.
```

Comment:
```suggestion
{VirtProductName} is currently available on x86-64 CPUs. Arm-based nodes are not yet supported.
```

I think "Virtualization" should be replaced with "{VirtProductName}" i.e OpenShift Virtualization